recipes-security/refpolicy/refpolicy-mcs-2.%/policy/modules/services/disman.fc

```diff
@@ -0,0 +1,6 @@
+/etc/init\.d/disman		--	gen_context(system_u:object_r:disman_initrc_exec_t,s0)
+
+/usr/bin/disman			--	gen_context(system_u:object_r:disman_exec_t,s0)
+/usr/bin/disman-hotplug.sh	--	gen_context(system_u:object_r:disman_exec_t,s0)
```

Comment:
Swith this to disman_script_exec_t:
/usr/bin/disman-hotplug.sh     --      gen_context(system_u:object_r:disman_script_exec_t,s0)

Reply:
... and also `\` that `.`.

Reply:
Yes. Good self-review 🙂